index.js

```diff
@@ -144,47 +179,60 @@ function nowServing(line) {
 }
 
 const line = []
+let line_position = 0
 
 // ***** Scope & Closures - Question 1 *****
-
+function takeATicketNumber(line) {
+  
+  if (!line.length) {
+    line_position++;
+  } else {
+    line_position = (line[line.length - 1] + 1);
+  }
```

Comment:
Here, you can just increment the line_position each time the function is called - no need for the extra conditional logic.